examples/fabric-admin/rpc/RpcClient.h

```diff
@@ -22,17 +22,19 @@
 
 #include "fabric_bridge_service/fabric_bridge_service.rpc.pb.h"
 
-constexpr uint16_t kFabricBridgeServerPort = 33002;
-
 /**
- * @brief Initializes the RPC client with the specified server port.
+ * @brief Sets the RPC server port to which the RPC client will connect.
  *
- * This function sets the RPC server port and starts packet processing for the RPC client.
+ * @param port The port number.
+ */
+void SetRpcClientPort(uint16_t port);
```

Comment:
SetRpcClientPort is confusing, since we don't have client port in RPC, we are setting the remote RPC server port to connect here